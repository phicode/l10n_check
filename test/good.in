# for checking the 0 return value on valid property files
a=b
