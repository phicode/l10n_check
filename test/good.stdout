test/good.in - 1 keys

test/good.in - no warnings or errors
